wire-ios-data-model/Source/Model/User/Availability/Availability.swift

```diff
@@ -16,13 +16,7 @@
 // along with this program. If not, see http://www.gnu.org/licenses/.
 //
 
-import Foundation
-import WireDataModel
-
-extension ZMUser {
-    func updateAvailability(_ newValue: AvailabilityKind) {
```

Comment:
This code was duplicate, see [ZMUser+Availability.swift](https://github.com/wireapp/wire-ios/pull/917/files#diff-8324f473a067274ac5ec7162e57e737adde4bba8358a2a81f2ed28d4c3d463f2R125)